onadata/apps/api/permissions.py

```diff
@@ -57,6 +57,30 @@ class ViewDjangoObjectPermissions(DjangoObjectPermissions):
     }
 
 
+class ExportDjangoPermission(IsAuthenticated,
```

Comment:
Can you rename this to `ExportDjangoObjectPermission` to be consistent with the others.